packages/cluster/src/common.js

```diff
@@ -80,8 +85,23 @@ function replacer(string, token, value) {
     return string.replace(regexp, value)
 }
 
-function resolveConfiguration(argv = {}, cache = {}, config = {}) {
-    const resolved = Object.assign({}, defaults, config, cache, argv)
+async function resolveConfiguration(argv = {}) {
+    const file = path.join(clusterDir, argv.name, cacheFile)
+    const currentCache = JSON.parse(await argv.getCache().read(file))
```

Comment:
This should be try-guarded or at least have a `fs.exists` check

Reply:
4f7d7d2